2022-01-15 18:47:06,945 - INFO - The Crypto-Copy-Trader starts launch
2022-01-15 18:47:06,946 - INFO - Reading configuration file...
2022-01-15 18:47:06,946 - INFO - 2 Slave accounts detected
2022-01-15 18:47:06,946 - INFO - Connecting to the master: Master...
2022-01-15 18:47:07,585 - ERROR - Error initialing exchanges
Traceback (most recent call last):
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\SlaveContainer.py", line 24, in __init__
    self.master = factory_method_create_exchange(config['master'], pairs)
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\SlaveContainer.py", line 14, in factory_method_create_exchange
    return necessary_class(single_config['key'], single_config['secret'], pairs, single_config['name'])
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\ExchangeInterfaces\BinanceExchange.py", line 29, in __init__
    self.update_balance()
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\ExchangeInterfaces\BinanceExchange.py", line 48, in update_balance
    account_information = self.connection.get_account()
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\venv\lib\site-packages\binance\client.py", line 1956, in get_account
    return self._get('account', True, data=params)
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\venv\lib\site-packages\binance\client.py", line 371, in _get
    return self._request_api('get', path, signed, version, **kwargs)
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\venv\lib\site-packages\binance\client.py", line 334, in _request_api
    return self._request(method, uri, signed, **kwargs)
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\venv\lib\site-packages\binance\client.py", line 315, in _request
    return self._handle_response(self.response)
  File "F:\1project\31.python_zignaly\me\workspace\binance_copy\venv\lib\site-packages\binance\client.py", line 324, in _handle_response
    raise BinanceAPIException(response, response.status_code, response.text)
binance.exceptions.BinanceAPIException: APIError(code=-1021): Timestamp for this request is outside of the recvWindow.
2022-01-15 18:47:07,594 - INFO - Open masters websocket... 
